src/pyclaw/solution.py

```diff
@@ -323,7 +323,9 @@ def read(self, frame, path='./_output', file_format='ascii',
          - *path* - (string) Base path to the files to be read. 
            ``default = './_output'``
          - *file_format* - (string) Format of the file, should match on of the 
-           modules inside of the io package.  ``default = 'ascii'``
+           modules inside of the io package.  If this is `None` then attempt to
+           determine format by file names and default to ``ascii`` if 
```

Comment:
Instead of "by file names" maybe we should say "by looking at the extensions of the files in the directory" (I think that's what is meant, right?)